CreateDataSetBlobTrigger/dataset_creator.py

```diff
@@ -0,0 +1,108 @@
+import inspect
+import logging
+import os
+import sys
+
+from datetime import datetime, timezone
+from dateutil import parser
+
+CURRENTDIR = os.path.dirname(
+    os.path.abspath(inspect.getfile(inspect.currentframe()))
+)
+PARENTDIR = os.path.dirname(CURRENTDIR)
+sys.path.insert(0, CURRENTDIR)
+sys.path.insert(0, PARENTDIR)
+
+from SharedCode.utils import get_cosmos_client, get_collection_link
+
+from SharedCode.exceptions import DataSetTooEarlyError
+
+
+class DataSetCreator:
+    def __init__(self):
+        self.cosmos_client = get_cosmos_client()
+        self.collection_link = get_collection_link(
+            "AzureCosmosDbDatabaseId", "AzureCosmosDbDataSetCollectionId"
```

Comment:
AzureCosmosDbDataSetCollectionId needs to be added to README.md at root level of repository

Reply:
I did update the top level README with this env variable plus others specific for this service.

Reply:
Sorry I missed the change in the README